contracts/src/v0.8/automation/dev/v2_3/AutomationRegistryLogicC2_3.sol

```diff
@@ -184,7 +184,10 @@ contract AutomationRegistryLogicC2_3 is AutomationRegistryBase2_3 {
       payments[i + activeTransmittersLength] = transmitter.balance;
       s_transmitters[deactivatedAddr].balance = 0;
     }
-    delete s_deactivatedTransmitters;
+
+    for (uint256 idx = 0; idx < s_deactivatedTransmitters.length(); idx++) {
+      s_deactivatedTransmitters.remove(s_deactivatedTransmitters.at(idx));
+    }
```

Comment:
same issue

Reply:
since s_deactivatedTransmitters is not exposed, i couldnt unit test this guy. I hope the above registrar test can also verify the logic.